frontend/views/containers/dashboard/GroupMembersActivity.vue

```diff
@@ -129,6 +122,38 @@ export default ({
             ? this.L('{user} missed {streak} votes', Largs)
             : this.L('{user} missed {streak} vote', Largs)
         })
+    },
+    memberCountSentences () {
+      const argsCommon = {
+        ...this.LTags('strong'),
+        'span_': '<span class="link t-trigger">',
+        '_span': '</span>'
+      }
+      const argsMap = {
+        'onTimePayments': { ...argsCommon, membercount: this.onTimePayments.length },
+        'haventLoggedIn': { ...argsCommon, days: this.config.notLoggedInDays, membercount: this.haventLoggedIn.length },
+        'noIncomeDetails': { ...argsCommon, membercount: this.noIncomeDetails.length },
+        'missedPayments': { ...argsCommon, membercount: this.missedPayments.length },
+        'noVotes': { ...argsCommon, membercount: this.noVotes.length, proposalcount: this.config.proposalNumber }
+      }
+
+      return {
+        'onTimePayments': this.onTimePayments.length === 1
+          ? this.L('{span_}{membercount} member{_span} have {strong_} on-time payment streaks{_strong}', argsMap['onTimePayments'])
+          : this.L('{span_}{membercount} members{_span} have {strong_} on-time payment streaks{_strong}', argsMap['onTimePayments']),
+        'haventLoggedIn': this.haventLoggedIn.length === 1
+          ? this.L('{span_}{membercount} member{_span} haven\'t {strong_} logged in past {days} days or more {_strong}', argsMap['haventLoggedIn'])
+          : this.L('{span_}{membercount} members{_span} haven\'t {strong_} logged in past {days} days or more {_strong}', argsMap['haventLoggedIn']),
+        'noIncomeDetails': this.noIncomeDetails.length === 1
+          ? this.L('{span_}{membercount} member{_span} haven\'t {strong_} entered income details{_strong}', argsMap['noIncomeDetails'])
+          : this.L('{span_}{membercount} members{_span} haven\'t {strong_} entered income details{_strong}', argsMap['noIncomeDetails']),
+        'missedPayments': this.missedPayments.length === 1
+          ? this.L('{span_}{membercount} member{_span} have {strong_} missed payments {_strong}', argsMap['missedPayments'])
+          : this.L('{span_}{membercount} members{_span} have {strong_} missed payments {_strong}', argsMap['missedPayments']),
+        'noVotes': this.noVotes.length === 1
+          ? this.L('{span_}{membercount} member{_span} haven\'t {strong_} voted in the last {proposalcount} proposals {_strong}', argsMap['noVotes'])
+          : this.L('{span_}{membercount} members{_span} haven\'t {strong_} voted in the last {proposalcount} proposals {_strong}', argsMap['noVotes'])
```

Comment:
Can you re-run the latest version of the `strings` utility on the `frontend/` folder to regenerate all of these strings?